src/bindings/glib/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ else()
 	target_compile_options (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_CFLAGS_OTHER})
 
 	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC elektra-core elektra-kdb)
-	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LDFLAGS})
+	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LIBRARIES})
```

Comment:
The difference of the two variables on my system:
```shell
-- Include Binding glib
CMake Warning at src/bindings/glib/CMakeLists.txt:38 (message):
  GOBJECT LIBRARIES: gobject-2.0glib-2.0


CMake Warning at src/bindings/glib/CMakeLists.txt:39 (message):
  GOBJECT LDFLAGS: -lgobject-2.0-lglib-2.0
```

the problem is the added -l when using ${GELEKTRA_LIBRARY} for the gtk doc as it get doubled and then not found eg `-l-lgobject-2.0`